src/constants.ts

```diff
@@ -23,3 +23,7 @@ export const GITHUB_PR_COMMENT_EDITOR_SELECTOR = "flex-nowrap d-inline-block mr-
 export const GITHUB_REPO_ACTIONS_SELECTOR = ".pagehead-actions";
 export const GITHUB_PR_COMMENT_TEXT_AREA_SELECTOR = "pull_request[body]";
 export const GITHUB_PR_BASE_BRANCH_SELECTOR = "css-truncate css-truncate-target";
+
+// External Links
+export const DISCUSSIONS = { link: "https://github.com/orgs/open-sauced/discussions", key: "Discussions", displayValue: "Go to feedback Discussions" };
```

Comment:
If you want it to be like this, maybe the following data structure would be better:

```
[
	{
		"key":"Discussions",
		"link":'http://....",
	},
	{
		"key":"Issues",
		"link":'http://....",
	}
]
```

Reply:
Yes, I agree with you on this.
THis would be the proper way of doing it instead of creating a new array in the page.

Reply:
Also, no need to add a display value here, especially one that includes "Go to.." since the external link icon basically means the same thing.

And Nice Work with this PR!